docs/sources/upgrade-guide/upgrade-v11.0/index.md

```diff
@@ -21,3 +21,9 @@ weight: 1200
 {{< docs/shared lookup="upgrade/upgrade-common-tasks.md" source="grafana" version="<GRAFANA VERSION>" >}}
 
 ## Technical notes
+
+### PostgreSQL data source plugin update
+
+<!-- Gabor Farkas -->
+
+If you use the PostgresSQL data source plugin and store certificates using the `Certificate content` method, Grafana previously creted temporary files in grafana's [data](/docs/grafana/latest/setup-grafana/configure-grafana/#data) folder. Grafana 11 does not use temporary files for this functionality anymore. But, existing temporary files will not be deleted, we recommend deleting them manually. To find them, look for the `tls` subfolder in the `data` folder, it contains subfolders named based on plugin ids, and these subfolders contain files named `client.*` or `root.*`. These are not used anymore, and can be deleted.
```

Comment:
Small typo: `creted`

`Grafana 11 does not use temporary files for this functionality anymore` - do you think it makes sense to include the reason being the change of lib/pq to pgx? I'm not entirely sure how much info we usually give with updates/changes.

To be clear, we are recommending users to delete the temp files because they're not needed by pgx anymore but if they don't delete them, it's not like anything will break?

Reply:
@vickyyyyyyy 

> Small typo: `creted`

thanks, fixed by @yesoreyeram 's change 👍 


> `Grafana 11 does not use temporary files for this functionality anymore` - do you think it makes sense to include the reason being the change of lib/pq to pgx? I'm not entirely sure how much info we usually give with updates/changes.

i'm not sure. the reason is somewhat complicated. it's partially related to the lib/pq to pgx change, in that, the pgx-change allows us to do this, but still, we could have stayed with the temp-files. but, not having temp-files is more robust, so we did some changes to be able to not have them...so, now that i'm reading this back, stuff is complicate, so i think it's better not to explain the whole thing in this document.. but i don't have strong opinions about this, so if you prefer me to add more explanations, i can do that, no problem.

> To be clear, we are recommending users to delete the temp files because they're not needed by pgx anymore but if they don't delete them, it's not like anything will break?

yes, correct.